src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/OffenderEmploymentRepository.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.justice.hmpps.prison.repository.jpa.repository;
+
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.PagingAndSortingRepository;
+import org.springframework.data.repository.query.Param;
+import uk.gov.justice.hmpps.prison.repository.jpa.model.OffenderEmployment;
+
+public interface OffenderEmploymentRepository extends PagingAndSortingRepository<OffenderEmployment, OffenderEmployment.PK> {
+
+    @Query(
+        value = """
+            SELECT OE
+            FROM OffenderBooking B
+                     INNER JOIN OffenderEmployment OE ON OE.id.bookingId = B.bookingId
+            WHERE B.offender.nomsId = :nomsId
```

Comment:
normally good to have some sorting on the returned results, don't see how paging would work properly otherwise